src/bundle/Resources/public/js/scripts/admin.notifications.list.js

```diff
@@ -0,0 +1,176 @@
+(function (global, doc, ibexa, Translator, Routing) {
+    const SELECTOR_MODAL_ITEM = '.ibexa-notifications-modal__item';
+    const SELECTOR_GO_TO_NOTIFICATION = '.ibexa-notification-view-all__show';
+    const SELECTOR_TOGGLE_NOTIFICATION = '.ibexa-notification-view-all__mail';
+    const { showErrorNotification } = ibexa.helpers.notification;
+    const { getJsonFromResponse } = ibexa.helpers.request;
+    const markAllAsReadBtn = doc.querySelector('.ibexa-notification-list__mark-all-read');
```

Comment:
Looking at the bottom two _querySelectors_ I would expect to have `ibexa-notification-list__btn--mark-all-as-read` class.